loaders/src/glTF/glTFFileLoader.ts

```diff
@@ -150,6 +150,11 @@ export class GLTFFileLoader implements IDisposable, ISceneLoaderPluginAsync, ISc
 
     private _onParsedObserver: Nullable<Observer<IGLTFLoaderData>>;
 
+    /**
+     * When loading GLTF Animations, which are defined in seconds, target them to this FPS.
+     */
+    public static TargetFps = 60;
+

```

Comment:
This is the old way of setting properties. It would be better to follow the pattern from the V2 options below.

Reply:
Updated to a non static variable!